model/src/main/scala/explore/data/tree/tree.scala

```diff
@@ -3,18 +3,29 @@
 
 package explore.data.tree
 
+import cats.kernel.Eq
 import monocle.macros.Lenses
 
-@Lenses final case class Tree[A](children: List[Node[A]])
-@Lenses final case class Node[A](value: A, children: List[Node[A]] = List.empty)
+@Lenses final case class Tree[A](children: List[Node[A]]) {
+  def map[B](f: A => B): Tree[B] =
+    Tree(children.map(_.map(f)))
+}
+@Lenses final case class Node[A](value: A, children: List[Node[A]] = List.empty) {
+  def map[B](f: A => B): Node[B] =
+    Node(f(value), children.map(_.map(f)))
+}
 
 object Tree {
   def empty[A]: Tree[A] = Tree(List.empty)
   def apply[A](children: Node[A]*): Tree[A] = Tree(children.toList)
+
+  def eqTree[A]: Eq[Tree[A]] = Eq.fromUniversalEquals
```

Comment:
Would it be better to request an `Eq[A]` and compare that way?

Reply:
Good, idea. Done.